models/intermediate/int_zuora__account_enriched.sql

```diff
@@ -61,20 +61,20 @@ account_totals as (
 
     select 
         account_id,
+        min(most_recent_payment_date) as most_recent_payment_date,
 
-        {% set sum_cols = ['refund_amount', 'discount_charges', 'tax_amount', 'invoice_amount', 'invoice_amount_home_currency', 'invoice_amount_paid', 'invoice_amount_unpaid'] %}
-        {% for col in sum_cols %}
-            sum({{ col }}) as total_{{ col }},
-        {% endfor %}
+        {% if var('zuora__using_credit_balance_adjustment', true) %}
+        min(most_recent_credit_balance_adjustment_date) as most_recent_credit_balance_adjustment_date,
```

Comment:
same here, wouldn't most recent be max?

Reply:
seems to be max based on the original logic too

Reply:
Ugh, yes. Thanks for spotting.